sdks/python/setup.py

```diff
@@ -168,6 +168,7 @@ def get_version():
     # [BEAM-5628] Beam VCF IO is not supported in Python 3.
     'pyvcf>=0.6.8,<0.7.0; python_version < "3.0"',
     # fixes and additions have been made since typing 3.5
+    'requests>=2.24.0',
```

Comment:
Using the current version as the minimum bound is fine. Bound this above at <3.0. 

https://requests.readthedocs.io/en/master/dev/philosophy/#semantic-versioning